### PR TITLE
Add and update tests for v6 migration

### PR DESCRIPTION
**Pull request by ajbozarth**
#172 was merged without updating the tests in order to cut rc1.
This PR include the required test updates.
 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

**Comment by ajbozarth** [comment]
Just pushed an update that makes the current set of tests pass, this included adding a mock of palette.

Still have TODOs for testing v5->v6 migration

**Comment by ajbozarth** [comment]
all the tests are now added/updated and ready for review

File: packages/pipeline-services/src/migration/utils/index.ts [hunk 1]
```diff
@@ -0,0 +1,5244 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// mock palette created by copying palette content sent to migrate function
+// and merging it into a single json object
+
+export const mockPalette: any = {
```
Comment (marthacryan) [inline packages/pipeline-services/src/migration/utils/index.ts:20]: Maybe these huge constants should go in a separate file, it's really hard to read it here

Comment (marthacryan) [reply packages/pipeline-services/src/migration/utils/index.ts:20]: Oh nevermind haha didn't realize that was the entire file

**Review by karlaspuldaro — APPROVED** [review]
LGTM

**Review by marthacryan — APPROVED** [review]
LGTM

**Pull request merged**